src/components/ContextBreadcrumbs.tsx

```diff
@@ -216,7 +219,12 @@ const ContextBreadcrumbs = ({
       - The "c/d" context will render "d" as a thought and "c" as the breadcrumbs.
     */
         !homeContext ? (
-          <HomeLink color='gray' size={16} style={{ position: 'relative', left: -1, top: 2 }} />
+          <HomeLink
+            className={css({ position: 'static' })}
+            color='gray'
+            size={16}
+            iconStyle={{ position: 'relative', left: -1, top: 2 }}
```

Comment:
This also has the object reference problem, although it's not as significant since `HomeLink` is rendered rarely, and is a very light component.